### ISSUE TITLE
Firefly character sheet not complete

### ISSUE THREAD

**Issue by sioc-de-narf** [Ramblurr/roll20-character-sheets#2]
Hello,

By looking at https://app.roll20.net/forum/post/900357/firefly-c... and https://app.roll20.net/forum/post/1102319/slug%7D the CortexPlus-DicePool.js is still missing in the github folder of firefly character sheet for a long time and that is making it unusable even with a mentor account.

Do we have a chance to get that js file or should we request the delation of that sheet ?

**Comment by Ysharros** [comment]
I would be super keen to know this too, but with no movement in months I expect the second solution is more likely. :(

**Comment by Ramblurr** [comment]
Hey folks! So sorry I haven't gotten on this. I'll put the file on github this week. I haven't used it in a about half a year, but it should still work if there haven't been any major API changes.

**Comment by Ysharros** [comment]
Awesome, thanks Casey! Yay!

**Comment by Ramblurr** [comment]
I tested it out this weekend, and there are some bugs do to new api changes. I'll fix them up and post it in this repo soon.

**Comment by Ysharros** [comment]
Thank you thank you! It'll be awesome to have the great-looking sheets working mechanically!

**Comment by sioc-de-narf** [comment]
Ramblurr : No News ?

**Comment by Ysharros** [comment]
Indeed. I have a group champing at the bit :D

**Comment by sioc-de-narf** [comment]
Hi,

Is that a try for vapoware in Roll20 world ?

**Comment by Ysharros** [comment]
That's not entirely fair. We don't pay for these things and the people who make these scripts aren't paid for doing them. Interests change, schedules get busy, and we simply forget.

I would be overjoyed if @Ramblurr finds the time to finish the project but I'll understand if they don't.

**Comment by sioc-de-narf** [comment]
Yes we are doing character sheet for free, that I agree, but I think that posting a not finished sheet is already not cool, if someone was thinking to do "I might have done one but one exist or is in progress".

It's nice to do sheets for everybody, but doing half the work and post it is worst than doing nothing.

Sorry if i sound bad and vindicative.

I am waiting for a working sheet with Cortex+ system for a long time and such waiting is just making me grunchy.

**Comment by Ysharros** [comment]
Heheh I understand, I've been waiting for ages too and was super happy to find one for FF/C+. Yes, it's disappointing.

@Ramblurr, I realise it's only two of us, but if you do find the time to do whatever coding magic is required to make these work the two of us would be super grateful. :)

**Comment by Ramblurr** [comment]
**TL;DR: [Here is the script file](https://github.com/Ramblurr/roll20-cortex-plus-engine/blob/master/dist/cortex-engine.js)**

Hey everyone! I am so sorry this took so long. I've put the project on github here: https://github.com/Ramblurr/roll20-cortex-plus-engine

It requires some graphics, and not all of them are included, because I don't have permission to distribute them yet. But the documentation is pretty extensive, so I hope everyone can get it up and running.

I haven't tested it in about a year, but it should work. I would love  to get bug fixes and such implemented if possible.

I will also update the documentation in the Firefly sheet to point there so people can get the latest version of the script.

The README.md has basic documentation, but for the full version, [download the repository](https://github.com/Ramblurr/roll20-cortex-plus-engine/archive/master.zip) and open the README.html file.

**Comment by Ysharros** [comment]
Thanks Casey!
